src/Symfony/Component/Form/Extension/Core/EventListener/ResizeFormListener.php

```diff
@@ -77,6 +80,11 @@ public function preSetData(FormEvent $event)
         }
     }
 
+    public function postSetData(PostSetDataEvent $event): void
+    {
+        $this->preSetData($event);
```

Comment:
this should be done the other way around: the deprecated method calls the non-deprecated one

Reply:
By doing the other way around we might break a child class by not calling `preSetData` anymore.
The last time I did such deprecation we handle it that way https://github.com/symfony/symfony/commit/ad08b951d798a5419f57bb092d636ff2c86dfece, https://github.com/symfony/symfony/commit/8a6cf9d93eda5d4c7685fd794a3733f787368055, should we do something similar instead?

Reply:
I guess so yes. The runtime notice is a must.

Reply:
See https://github.com/symfony/symfony/pull/51041/commits/ba36dd37091dd6df8b7953185c4bfc0f1f1bf920.